examples/helloworld/torch_llm/README.md

```diff
@@ -0,0 +1,15 @@
+Applying Weight-Only Quantization to LLMs
+=====================
+This example demonstrates how to apply weight-only quantization on [Mistral-7B](https://huggingface.co/mistralai/Mistral-7B-v0.1).
+
+
+### Setting up the environment 
+```shell
+pip install -r requirements.txt
+```
+
+### Running the Quantization Script
+
+```shell
+python quant_mistral.py
```

Comment:
the script name should align with tf examples.

Reply:
It appears that `quant_mistral.py` is more meaningful than `test.py`. :)